cedar-lean/Cedar/Spec/Wildcard.lean

```diff
@@ -46,7 +47,7 @@ def wildcard : PatElem → Bool
   | .star => true
   | _     => false
 
-abbrev Cache := HashMap (Nat × Nat) Bool
+abbrev Cache := Std.HashMap (Nat × Nat) Bool
```

Comment:
Do you know why we have to write `Std.HashMap` here even though we have `open Std` at the top?

Would be nicer to have just `HashMap`.